docs/flinkDev/building.md

```diff
@@ -127,8 +127,7 @@ mvn clean install -Dhadoop.version=2.6.5-custom
 
 After this step is complete, follow the steps for [Pre-bundled versions](#pre-bundled-versions).
 
-To build Flink against a vendor specific Hadoop version, additionally activate -Pvendor-repos` profile when building
-`flink-shaded`.
+To build Flink against a vendor specific Hadoop version, additionally `activate -Pvendor-repos` profile when building flink-shaded:
```

Comment:
The tick here seems to be in the wrong position, still.

Reply:
Oh, my god! Sorry, this is my fault.